File: backend/project2/src/main/java/com/team8/project2/domain/member/dto/FollowResDto.java
package com.team8.project2.domain.member.dto;

import com.team8.project2.domain.member.entity.Follow;
import lombok.Getter;

import java.time.LocalDateTime;

@Getter
public class FollowResDto {

	private String followee;
	private String profileImage;
	private LocalDateTime followedAt;

	public static FollowResDto fromEntity(Follow follow) {
		FollowResDto followResDto = new FollowResDto();
		followResDto.followee = follow.getFolloweeName();
		followResDto.profileImage = follow.getFolloweeProfileImage();
		followResDto.followedAt = follow.getFollowedAt();
		return followResDto;
	}
}

File: backend/project2/src/main/java/com/team8/project2/domain/member/dto/FollowResDto.kt
package com.team8.project2.domain.member.dto

import com.team8.project2.domain.member.entity.Follow
import java.time.LocalDateTime

class FollowResDto(
    val followee: String?,
    val profileImage: String?,
    val followedAt: LocalDateTime?
) {
    companion object {
        @JvmStatic
        fun fromEntity(follow: Follow): FollowResDto {
            return FollowResDto(
                follow.followeeName,
                follow.followeeProfileImage,
                follow.followedAt
            )
        }
    }
}